### PR TITLE
Add daily task to scan topics

### PR DESCRIPTION
**Pull request by szakarias**

### CODE REVIEW

File: app/lib/tool/neat_task/pub_dev_tasks.dart [hunk 1]
```diff
@@ -157,6 +157,8 @@ void _setupGenericPeriodicTasks() {
     ),
   );
 
+  _daily(name: 'topics', isRuntimeVersioned: false, task: countTopics);
```
Comment (jonasfj) [inline app/lib/tool/neat_task/pub_dev_tasks.dart:160]:
```suggestion
  _daily(name: 'count-topics', isRuntimeVersioned: false, task: countTopics);
```

Comment (jonasfj) [reply app/lib/tool/neat_task/pub_dev_tasks.dart:160]: @isoos there is some try-catch sanity around this, right?

**Review by jonasfj — APPROVED** [review]

File: app/lib/tool/neat_task/pub_dev_tasks.dart (hunk 1 above)
Comment (isoos) [reply app/lib/tool/neat_task/pub_dev_tasks.dart:160]: Yeah, it will setup it in a sane way.

Comment (szakarias) [reply app/lib/tool/neat_task/pub_dev_tasks.dart]: done

**Pull request merged**